src/darwin/Framework/CHIP/templates/MTRClusterNames-src.zapt

```diff
@@ -75,6 +75,62 @@ NSString * MTRAttributeNameForID(MTRClusterIDType clusterID, MTRAttributeIDType
                 result = [NSString stringWithFormat:@"<Unknown attributeID %d>", attributeID];
                 break;
         }
+        break;
```

Comment:
Oh, nice catch!

Reply:
Added a test in https://github.com/project-chip/connectedhomeip/pull/34687 that would have caught this.